lms/services/jwt.py

```diff
@@ -86,15 +86,22 @@ def decode_lti_token(self, id_token: str) -> dict:
         if not id_token:
             return {}
 
-        if not jwt.get_unverified_header(id_token).get("kid"):
+        try:
+            unverified_header = jwt.get_unverified_header(id_token)
+            unverified_payload = jwt.decode(
+                id_token, options={"verify_signature": False}
+            )
+        except PyJWTError as err:
+            LOG.debug("Invalid JWT. %s", str(err))
```

Comment:
```suggestion
            LOG.exception("Invalid JWT")
```

You can instead do `LOG.debug("Invalid JWT", exc_info=True)` and it'll find the current exception for you automatically and log it properly (with a full traceback including exception chaining). Or, probably better, `LOG.exception("Invalid JWT")` (you don't even need `exc_info=True`) and again it'll log the exception properly with the stack trace, this time with log level `ERROR`.